src/util/Compiler.h

```diff
@@ -21,8 +21,12 @@ Copyright_License {
 }
 */
 
-#ifndef COMPILER_H
-#define COMPILER_H
+#ifndef UTIL_COMPILER_H
+#define UTIL_COMPILER_H
+
+#ifdef _MSC_VER
+#   include "util/msvc"
```

Comment:
Missing `.h`

Reply:
Of course!
I don't know, how this happens ;-(